test/avro_tests.py

```diff
@@ -344,3 +344,25 @@ class ValidRecord(pyschema.Record):
         line = '{"field": {"long": 8}, "invalid_field": {"long": 8}}'
 
         self.assertRaises(ParseError, lambda: pyschema_extensions.avro.loads(line, schema=ValidRecord))
+
+
+class TestNamespaceMigration(TestCase):
+    def test_old_data(self):
+        """
+        Test that we are able to read data created without namespace after namespace is added.
+        """
+        class NamespacedSubRecord(Record):
```

Comment:
I need to do it in order to test the parsing when not including schema in the call to pyschema_extensiions.avro.loads

That test might not be so relevant however